src/pages/Share/ShareTab.tsx

```diff
@@ -97,19 +99,20 @@ function ShareTab({ref}: ShareTabProps) {
         searchInServer(debouncedTextInputValue.trim());
     }, [debouncedTextInputValue]);
 
-    const styledRecentReports = recentReportsOptions.map((item) => ({
-        ...item,
-        pressableStyle: styles.br2,
-        text: StringUtils.lineBreaksToSpaces(item.text),
-        wrapperStyle: [styles.pr3, styles.pl3],
-    }));
-
-    const [sections, header] = useMemo(() => {
-        const newSections = [];
-        newSections.push({title: textInputValue.trim() === '' ? translate('search.recentChats') : undefined, data: styledRecentReports});
+    const styledRecentReports = useMemo(() => {
+        return recentReportsOptions.map((item, index) => ({
+            ...item,
+            pressableStyle: styles.br2,
+            text: StringUtils.lineBreaksToSpaces(item.text),
+            wrapperStyle: [styles.pr3, styles.pl3],
+            keyForList: String(item.reportID) + String(index),
```

Comment:
```suggestion
            keyForList: `${item.reportID}-{index},
```
Won't it work like that?